crates/core/tedge_api/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
```

Comment:
Is this part of the proposal? Otherwise, could you remove the extraneous parts so that it is clear what exactly the proposal contains?

Reply:
This is missing a few commits, including the clean up of extraneous packages.
I'll update this PR this week to add missing elements as well as adding missing docs.